about.html

```diff
@@ -0,0 +1,162 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <header>
+      <div class="nav-bar1">
+        
+          <a href="facebook.com/promzzyokechukwu"><i class="fa-brands fa-facebook-f"></i></a>
+          <a href="facebook.com/promzzyokechukwu"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#">Logout</a>
+          <a href="https://promzzykoncepts.github.io/My-Portfolio/">My Page</a>
+          <a href="#">English</a>
+        </div>
+      
+      <nav class="nav-bar">
+
+        
+      <ul class="nav-menu">
+        <li class="nav-item logo">
+          
+          <a href="#"><img class="logo-img" src="images/R.png" alt="logo"/> 
+          <h3>REMOTE GLOBAL  SUMMIT<br>2022</h3></a>
+          
+        </li>
+          <li class="nav-item">
+              <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Join</a>
+          </li>
+             <li class="nav-item labelT">
+            <a href="" class="nav-link">WFH Remoters..</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+          
+      </div>
+      </nav>
+  </header>
+
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH Work from Home"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             
+             </div>
+             <div class="mid-label">
+              <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/R.png" alt="logo"/></a>
+              </div>
+             <div class="footer-content">
```

Comment:
- [ ] For the logo section I think the width for the image is not right here and it doesn't look like the original design so kindly fix to be the same since it is one of the reuirements 

- [ ] As I can see you have the image first then the text but in the original design the opposite so kindly add the text first then the logo image  to make your website nicer 😄

- Your design 
![image](https://user-images.githubusercontent.com/58553711/183261216-2b2a6acb-ee33-4ab0-8a08-381aa3d5a0cb.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/183261250-5ea36ccd-96b0-4b5e-b529-d097695d8ae5.png)